docs/content/getting_started/additionals.md

```diff
@@ -30,12 +31,12 @@ export DATABRICKS_TOKEN=<DATABRICKS TOKEN> # Account needs permission to create
         3. Using Databricks CLI, Create databricks secrets to store producer and consumer keys using the scope created in step 2 
 
         Following are the mandatory arguments for running EventHubs integration test
-        1. Provide your eventhub topic name : ```--eventhub_name```
-        2. Provide eventhub namespace using ```--eventhub_namespace```
-        3. Provide eventhub port using ```--eventhub_port```
-        4. Provide databricks secret scope name using ```----eventhub_secrets_scope_name```
-        5. Provide eventhub producer access key name using ```--eventhub_producer_accesskey_name```
-        6. Provide eventhub access key name using ```--eventhub_consumer_accesskey_name```
+        1. Provide your eventhub topic : --eventhub_name
```

Comment:
it's better to put the code-like text into single backticks

Reply:
Done!